scimma_admin/hopskotch_auth/templates/hopskotch_auth/login_failure.html

```diff
@@ -0,0 +1,7 @@
+{% extends "hopskotch_auth/base.html" %}
+
+{% block page-body %}
+    Login Failure
```

Comment:
Maybe make this a bit louder? I think this will be some relatively quiet text, as-is. Maybe a bit of `<h3>` or something?

Eh, who am I kidding, I don't know how to UI.

Reply:
Sure `<h3>` it is.